### ISSUE TITLE
[semi-urgent] Footer display in dsahboards is fucked up

### ISSUE THREAD

**Issue by NunoSempere** [QURIresearch/metaforecast#45]
See e.g., https://metaforecast.org/dashboards?dashboardId=ad190968c6, or the dashboards embedded in https://globalguessing.com/russia-ukraine-forecasts/

**Comment by berekuk** [comment]
Oops. I'll fix it in 1-2 hours.

**Comment by berekuk** [comment]
Fixed in 3771b53e4fe391cc32eba1497c6c343c9974d1cc.

There are two issues here.

One is that I fucked up platform labels by not being careful with adding labels in all cases. So dashboard cards displayed `goodjudgmentopen` instead of `GJOpen`. There are currently two types, `Forecast` and `FrontendForecast`; I changed the type signature of `<DisplayForecasts>` component in my previous commit, but unfortunately haven't got around to changing types for `/dashboards` page, and since `any` type is allowed and most return values are still `any`, typescript haven't caught this, and I wasn't careful enough.

Second issue is that CSS could accomodate this situation better. I'll look into CSS now before closing this issue, to avoid later complications.
